### PR TITLE
Login

### PR DESCRIPTION
**Pull request by praneetmek**
Changed API

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -5,8 +5,13 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://apis.google.com/js/platform.js" async defer></script>
+  <meta name="google-signin-client_id" content="757099697912-i6jll98mfgochdo2vgjcovf64pepjesc.apps.googleusercontent.com">
```
Comment (geoffreyzgoog) [inline src/main/webapp/index.html:9]: Where is the content URL from?

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -5,8 +5,13 @@
...
+
+
   <script type="module" src="scripts/index-script.js" defer></script>
-  <link rel="stylesheet" href="styles/general.css">
```
Comment (geoffreyzgoog) [inline src/main/webapp/index.html:9]: Is this not needed any more for consistent styling?

File: src/main/webapp/index.html (hunk 1 above)
Comment (praneetmek) [reply src/main/webapp/index.html:9]: I got that from the gcp dashboard, under the api credentials section

File: src/main/webapp/index.html (hunk 2 above)
Comment (praneetmek) [reply src/main/webapp/index.html:9]: I honestly dont remember deleting that, but i just ran it on a server and all the styling seems to be fine?

File: src/main/webapp/index.html (hunk 1 above)
Comment (geoffreyzgoog) [reply src/main/webapp/index.html:9]: I see, can you add a comment for that?

Comment (praneetmek) [reply src/main/webapp/index.html:10]: added

**Pull request merged**